examples/encoding/drm_integrations/create_drm_encoding_with_verimatrix.py

```diff
@@ -0,0 +1,389 @@
+import json
+import requests
+import datetime
+from bitmovin import Bitmovin, Encoding, S3Output, H264CodecConfiguration, \
+    AACCodecConfiguration, H264Profile, StreamInput, SelectionMode, Stream, EncodingOutput, ACLEntry, ACLPermission, \
+    FMP4Muxing, MuxingStream, DashManifest, DRMFMP4Representation, FMP4RepresentationType, Period, \
+    VideoAdaptationSet, AudioAdaptationSet, ContentProtection, HTTPSInput, HlsManifest, VariantStream, \
+    AudioMedia, FairPlayDRM, TSMuxing, AWSCloudRegion
+from bitmovin import CENCDRM as CENCDRMResource
+from bitmovin.resources.models import CENCPlayReadyEntry, CENCWidevineEntry
+from bitmovin.errors import BitmovinError
+
+def main():
+
+    API_KEY = '<INSERT_YOUR_API_KEY>'
+
+    # https://<INSERT_YOUR_HTTP_HOST>/<INSERT_YOUR_HTTP_PATH>
+    HTTPS_INPUT_HOST = '<INSERT_YOUR_HTTPS_HOST>'
+    HTTPS_INPUT_PATH = '<INSERT_YOUR_HTTPS_PATH>'
+
+    S3_OUTPUT_ACCESSKEY = '<INSERT_YOUR_ACCESS_KEY>'
+    S3_OUTPUT_SECRETKEY = '<INSERT_YOUR_SECRET_KEY>'
+    S3_OUTPUT_BUCKETNAME = '<INSERT_YOUR_BUCKET_NAME>'
+
+    date_component = str(datetime.datetime.now()).replace(' ', '_').replace(':', '-').split('.')[0].replace('_', '__')
+    OUTPUT_BASE_PATH = '/encoding/{}/'.format(date_component)
+
+    contentId = "<YOUR_VERIMATRIX_CONTENT_ID>"
+    siteId = "<YOUR_VERIMATRIX_SITE_ID>"
+
+    contentURL = 'http://your.verimatrix.endpoint:8058/cei/v1.0/content/{}?siteId={}'
+    contentURL = contentURL.format(contentId, siteId)
+
+    contentRequest = """{
+        "contentRequest": {
+            "siteId": "",
+            "contentId": "",
+            "contentType": "VOD",
+            "streamingProtocol": "SS",
+            "encryptionAlgorithm": "AES_CTR",
+            "keyRotation": {
+                "chainedLicense": "FALSE",
+                "cryptoPeriod": "0"
+            }
+        }
+    }"""
+
+    parsed_json = json.loads(contentRequest)
+    parsed_json["contentRequest"]["contentId"] = contentId
+    parsed_json["contentRequest"]["siteId"] = siteId
+
+    headers = {'Content-Type': 'application/json'}
+
+    r = requests.put(contentURL, data=json.dumps(parsed_json),headers=headers)
```

Comment:
in general I'd put the content request into it's own method so it's more obvious what does what.. 

so probably a `def request_content` and a `def request_key` would help here..